code/modules/vehicles/mecha/mecha_mob_interaction.dm

```diff
@@ -42,13 +42,21 @@
 		return FALSE
 	newoccupant.drop_all_held_items()
 	add_occupant(newoccupant)
-	newoccupant.forceMove(src)
+	if(newoccupant.loc != src)
+		newoccupant.forceMove(src)
 	newoccupant.update_mouse_pointer()
```

Comment:
???

Reply:
This was causing repeat forcemove calls which caused issues with another change. I think minimap stuff.